src/npc_attack.cpp

```diff
@@ -19,9 +19,9 @@ const std::map<Creature::Attitude, int> attitude_multiplier = {
     { Creature::Attitude::NEUTRAL, -1 }
 };
 // if you are attacking your target, multiply potential by this number
-const int target_modifier = 3;
+const float target_modifier = 1.5f;
 // if you kill the creature, multiply the potential by this number
-const int kill_modifier = 2;
+const float kill_modifier = 1.5f;
```

Comment:
why are we switching to float math?

Reply:
It's a change to use small multipliers to not boost the priority too much for targets/kill targets. This can be reverted.